### ISSUE TITLE
pybullet renderImage m_depthBuffer is upsidedown

### ISSUE THREAD

**Issue by matpalm** [bulletphysics/bullet3#692]
at first i thought it'd be as simple as just including `flip_vertically` (as is done for the `m_rgbColorBuffer`) but i see now that `m_depthBuffer` isn't a `TGAImage` so i'm unsure what helper to use to do this flip...

there's a simple workaround though; flipping after the render...
```
_, _, _, depth = pybullet.renderImage(...)
numpy.flipud(depth)
```

**Comment by erwincoumans** [comment]
Is that an interview question? Attached is my untested whiteboard implementation, please test :-)

[swap_depth.zip](https://github.com/bulletphysics/bullet3/files/357655/swap_depth.zip)

**Comment by matpalm** [comment]
haha. not an interview question. was thinking that would be likely already be available in some helper function i didn't know about, but i guess not then :)

**Comment by erwincoumans** [comment]
Heh, the btAlignedObjectArray<> is a small subset of std::vector, no kitchen-sink features.
Does my patch work for you?

**Comment by matpalm** [comment]
I check first thing tomorrow!

**Comment by matpalm** [comment]
fixed in @6c856d9fa3865d990c9afcad02d4f97fea8aafa6
